testsuite/src/it/java/io/quarkus/ts/startstop/utils/WhitelistLogLines.java

```diff
@@ -77,7 +77,24 @@
             // (no explicit configuration, none or multiple JDBC driver extensions)
             // Result of DevServices support https://github.com/quarkusio/quarkus/pull/14960
             Pattern.compile(".*Unable to determine a database type for default datasource.*"),
+    }),
+    // Quarkus is not being gratefully shutdown in Windows when running in Dev mode.
+    // Reported by https://github.com/quarkusio/quarkus/issues/14647.
+    WINDOWS_DEV_MODE_ERRORS(new Pattern[]{
+            Pattern.compile(".*Re-run Maven using the -X switch to enable full debug logging.*"),
+            Pattern.compile(".*For more information about the errors and possible solutions, please read the following articles.*"),
+            Pattern.compile(".*Failed to run: Dev mode process did not complete successfully.*"),
+            Pattern.compile(".*http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException.*"),
+            Pattern.compile(".*To see the full stack trace of the errors, re-run Maven with the -e switch.*"),
+            Pattern.compile("\\[ERROR\\] *"),
     });
+	
+	// Randomly fails when vertx-cache temporary directory exists. Related to https://github.com/quarkusio/quarkus/issues/7678
+    // And https://github.com/quarkusio/quarkus/pull/15541/files#diff-a38e0d86cf6a637c19b6e0a0e23959f644886bdcc0f0e5615ce7cfa0e6bc9909R244
+	private static final Pattern COMMON_WARNING_VERTX_CACHE_DIRECTORY = Pattern.compile(".*Unable to make the Vert.x cache directory.*");
```

Comment:
Are these 3 really happening on all the platforms or just Windows?

Reply:
As I could see in the PR, it could happen in any platform.

Reply:
And does it happen? TBH it should not happen at all as a random directory is generated every time.

Reply:
It happened to me when trying on Windows. However, I will remove it and let's see.

Reply:
Changed to be only for Windows and reported.